source/_patterns/02-molecules/blocks/alert/alert~error.yml

```diff
@@ -0,0 +1,8 @@
+alert:
+  type: "error"
+  heading: "Error status"
+  text: "Lorem ipsum dolor sit amet, <a href=\"javascript:void(0);\">consectetur adipiscing</a> elit, sed do eiusmod."

```

Comment:
Hi @melwong-jcc!  Hope you don't mind the review, but just wanted to let you know that the `text` and `button` keys are not necessary in the variant data files. Only the data that actually changes is needed, the rest will be inherited from the base `alert.yml` data.

Reply:
I'm good with this suggestion. I was following another pattern to id a standard and probably did not notice the subtle changes in the values though they look duplicated. Will correct shortly.

Reply:
Thanks so much! 😁